### PR TITLE
[CARBONDATA-1451] Removing configuration for number_of_rows_per_blocklet_column_page

### PR DESCRIPTION
**Pull request by dhatchayani**

### CODE REVIEW

**Comment by ravipesala** [comment]
SDV Build Fail , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/563/

File: core/src/main/java/org/apache/carbondata/core/constants/CarbonV3DataFormatConstants.java [hunk 1]
```diff
@@ -61,24 +61,8 @@
...
-  /**
-   * number of rows per blocklet column page max value
-   */
-  short NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_MAX = 32000;
-
-  /**
-   * number of rows per blocklet column page min value
-   */
-  short NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_MIN = 8000;
+  short NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_DEFAULT = 32000;
```
Comment (jackylk) [inline core/src/main/java/org/apache/carbondata/core/constants/CarbonV3DataFormatConstants.java]: I do not think it is configurable, better to remove it from this file

Comment (dhatchayani) [reply core/src/main/java/org/apache/carbondata/core/constants/CarbonV3DataFormatConstants.java]: this is not configurable. 32000 is the default constant stored as "NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_DEFAULT"

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**